HostDependencies.md

```diff
@@ -97,15 +131,15 @@ The python dependencies required by the CAmkES build toolchain can be installed
 pip install --user camkes-deps
 ```
 
-### Haskell Dependencies
+### Haskell Dependencies (Duplicated)
```

Comment:
I'm not sure what the "Duplicated" infers here?

Reply:
I dont remember, it's too far away. Mabe it's duplicated with the camekes doc?

Reply:
Best to remove it then, thanks.